src/DataViews/include/DataViews/DataViewTypes.h

```diff
@@ -1,9 +1,11 @@
-// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
Consider renaming the file to `DataViewType.h` to match the name of the `enum`.

Reply:
Yes, good point. Done.